cmd/mt-gateway/main.go

```diff
@@ -17,6 +17,7 @@ var (
 	graphiteURL   = flag.String("graphite-url", "http://localhost:8080", "graphite-api address")
 	importerURL   = flag.String("importer-url", "", "mt-whisper-importer-writer address")
 	addr          = flag.String("addr", ":80", "http service address")
+	defaultOrgId  = flag.String("default-org-id", "", "default org ID to send to downstream services if none is provided")
```

Comment:
our org ids are all numeric, we might want to enforce that by using a `flag.Int` instead

Reply:
-1 as the default (which would be invalid, I assume)?

Reply:
sounds good. actually, i don't think MT would reject the org id of `-1`, it would just convert it into a `uint32` on ingest, resulting in some really high number. but there is no reason why we'd need to support negative numbers afaik

Reply:
d442d3de6a4e640f9ece61bb010967f3164685c3